Docker.md

```diff
@@ -0,0 +1,33 @@
+# Run the Darkflame Server inside Docker
+
+## What you need
+
+- Docker (Docker Desktop or on Linux normal Docker)
+- Docker-Compose (Included in Docker Desktop)
+- LEGO® Universe Client
+
+## Run server inside Docker
+
+1. Copy `.env.example` and save it as `.env` inside the root directory of this repository
+2. Edit the `.env` file and add your path to your LEGO® Universe Client after `CLIENT_PATH=`
+3. Update other values in the `.env` file as need (be sure to update passwords!)
+4. Run `docker compose up -d --build`
```

Comment:
I changed this and the other commands from `docker-compose` to `docker compose`. I think the Docker Desktop includes `docker compose` ([the v2 version](https://docs.docker.com/compose/cli-command/#:~:text=Docker%20Compose%20V2%20will%20be,version%20of%20the%20Docker%20CLI.)) by default now.

Reply:
Hmm I'm running Docker Desktop on WSL2 in Windows and it's `docker-compose`, I also run Docker on a Ubuntu server and it's the same. Same for my Mac at work.

Reply:
Basically using `docker-compose` is backwards compatible so I think we should keep using that.

Reply:
I'm good to leave it as is. I tested on all my clients and both ways work. I updated the docs with a note just in case someone is running an old version.